contributors/guide/code-review.md

```diff
@@ -0,0 +1,275 @@
+your code.
+
+For more guidance on commit messages, see our [commit message guidelines].
+
+
+### Pull Request comments should summarize the PR
+
+The GitHub pull request should contain a summary of the changes being made as
+well as links to all the relevant information related to the PR itself. These
+are things such as issues it closes, KEPs, and any other relevant documentation.
+
+Accurately filling out the issue template and applying the necessary labels
+will greatly speed up the initial triage of your PR.
+
+If there is a portion of your PR that should be explained or linked that is not
+committed as a comment, you can review your own PR, applying additional context
+or links.
+
+
+### Minor changes (typos / link fixes)
+
+While these sort of fixes are appreciated, try and fix multiple instances at the
+same time if possible, instead of issuing subsequent individual PRs for each fix.
+
+---
+
+## Tips for code reviewers
+
+### Managing time
+
+#### Block time off to review
+
+Often it can be hard to find dedicated, uninterrupted time to review PRs. If you
+can, allocate some time and block it off on your calendar to help stay on top of
+the incoming queue.
+
+#### Taking a break or stepping away
+
+If you are taking a break, going on vacation, or stepping away for a bit -- you
+can set your [GitHub status] to busy; this will signal [Blunderbuss] to not
+automatically assign you to reviews (It does not block manual assignment).
+
+If for an extended period of time, or if you need to focus on other areas,
+consider setting yourself as an `[emeritus_approver]` in some of the areas your
+an approver.
+
+
+### Loop in others when Domain Specific Knowledge is needed
+
+Kubernetes has an incredibly large and complex code base with sections that may
+require domain-specific knowledge. If you are unsure or uncomfortable reviewing
+a portion of a PR, it is better to decline a review and reassign to an owner or 
+contributor with more expertise in that area.
+
+If you are brought in for your knowledge in a specific area, try and provide
+meaningful comments to serve as breadcrumbs in the future to help others gain a
+better understanding of that area.
+
+
+### Asking questions
+
+You are encouraged to ask questions and seek an understanding of what the PR is
+doing; however, your question might be answered further into the review. You can
+stage your questions, and before you submit your review, revisit your own comments
+to see if they're still relevant or update them after gaining further context. 
+
+Often a question may turn into a request for further comments or changes to
+explain what is happening at that specific point.
+
+In your questions, try and be empathetic when phrasing. Instead of:
+
+_"Why did you do this?"_
+
+try
+
+_"Am I understanding this correctly? Can you explain why...?"_
+
+Remember a review is a discussion, often with multiple parties -- be reasonable.
```

Comment:
"Be reasonable" feels ambiguous and less than actionable.  Perhaps something like "Try to focus and summarize in ways which constructively move the conversation forward instead of retreading ground."

Reply:
added 👍